scipy/stats/stats.py

```diff
@@ -1240,14 +1340,8 @@ def skewtest(a, axis=0, nan_policy='propagate'):
     -----
     The sample size must be at least 8.
 
-    References
-    ----------
-    .. [1] R. B. D'Agostino, A. J. Belanger and R. B. D'Agostino Jr.,
-            "A suggestion for using powerful and informative tests of 
-            normality", American Statistician 44, pp. 316-321, 1990.
```

Comment:
Looks like I missed https://github.com/scipy/scipy/commit/22f34db6420dc3f723edce14d76998886b160b14, adding it back in now